packages/utilities/package.json

```
@@ -43,9 +43,9 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/dom-utilities": "^2.0.0-beta.2",
-    "@fluentui/merge-styles": "^8.0.0-beta.5",
-    "@fluentui/set-version": "^8.0.0-beta.2",
+    "@fluentui/dom-utilities": "^2.0.0beta.2",
```

```suggestion
    "@fluentui/dom-utilities": "^2.0.0",
```

This looks suspicious 🐱 I think that it should be just `^2.0.0`...

---

oops, fixed